### DISCUSSION TITLE
interfaccia per settare temperatura desiderata.

### DISCUSSION THREAD

**Discussion by wyrden79** [indomus/forum#2049] [HUB: Home Assistant]
buongiorno a tutti, ho creato due automazioni per accendere i termosifoni quando in camera si scende sotto i 18.5 e si spengono quando in camera salgono sopra i 19, allego i codici
![termosifoni_accesi](https://user-images.githubusercontent.com/87671992/200289355-de9e2173-3bf2-4209-a85b-909131a28a2a.jpg)
![termosifoni_spenti](https://user-images.githubusercontent.com/87671992/200289828-88e41c48-8c5b-4b2c-a942-2249cb2f37a4.jpg)
vorrei creare una interfaccia su home della plancia dove possa modificare i gradi e in automatico si aggiornano le automazioni.
per esempio mettendo 20 in automatico si aggiorna con 20 l'automazione che spegne i termosifoni e 19.5 per accenderli.
grazie per chiunque mi possa aiutare.

**Comment by KC-inDomus** [comment]
in primis, è completamente sbagliato l'approccio delle automazioni. il trigger non deve essere configurato usando `time_pattern`, ma semmai `numeric_state` sul sensore termico.
vedi [qui](https://indomus.it/formazione/esercizi-di-domotica-su-home-assistant-2-definire-diversi-trigger-per-innescare-automazioni/#esercizio-2-3)

**Reply by snakuzzo** [reply]
...se ho capito bene...

perché fare delle automazioni se l'enittà di tipo `climate` fa già quello che ti serve ?
IMHO non hanno senso quelle automazioni.
Premesso quello che ti ha già scritto @KC-inDomus, se serve da GUI fare quello che dici per settare la temperatura, ti basta un input number per settare la temperatura del climate...ma ripeto che secondo me non ha senso...visto che se esponi il climate sulla GUI, puoi modificare la temperatura direttamente lì
